### ISSUE TITLE
Possible problem with table cells and their headers

### ISSUE THREAD

**Issue by SjurSutterudSagen** [PxTools/pxtools-project#201]
The Mabl smoketest is failing on the last accessibility check step, with the message: 

<img width="1624" height="564" alt="Image" src="https://github.com/user-attachments/assets/deda3856-c63b-4674-8185-6a7dc6934179" />

This might be because we are adding the IDs of different cells into the headers attribute on the table cells, but some of those cells might not be loaded in the html in the new virtualized table.

I am not even sure this is a problem with screenreaders, but it is something to check atleast.

**Comment by SjurSutterudSagen** [comment]
Asked Copilot about it, and got this:

<img width="824" height="379" alt="Image" src="https://github.com/user-attachments/assets/8b0e6a54-84c3-495e-ba0c-d201ef8dd363" />

**Comment by Maren-SSB** [comment]
@SjurSutterudSagen Vi har denne også: https://github.com/PxTools/pxtools-project/issues/136 Er vel ikke det samme, men kanskje funnene ville vært de samme?

**Comment by SjurSutterudSagen** [comment]
I ran the html-validate step, but I didn't look like this was picked up in those results.
